general/calendars/outlook/outlook_calendar.rst

```diff
@@ -0,0 +1,57 @@
+=====================================
+Synchronization with Outlook Calendar
+=====================================
+
+Life is a lot easier when you are able to manage all your calendars in one single place. Therefore,
+now Odoo also allows you to synchronize your Outlook Calendar with the Calendar application.
+
+To start, `Register your application with the Microsoft identity platform
+<https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app>`_ making
+sure to **follow all the steps described in the guidelines**.
+
+.. important::
+   For the **Redirect URI restrictions**, copy your Odoo server URI followed by
+   ``/microsoft_account/authentication``. Example:
+
+   .. image:: media/redirect_URIs.png
+      :align: center
+      :alt: Odoo's server URI that is accepted when microsoft returns authentication
+
+Once you are finished, copy:
+
+- Client ID:
+
+.. image:: media/application_client_ID.png
+   :align: center
+   :alt: Client ID token to be copied from Microsoft
+
+- Client Secrets:
+
+.. image:: media/client_secrets.png
+   :align: center
+   :alt: Client Secret token to be copied from Microsoft
```

Comment:
more explanation with a `:menuselection:` would help the users to find this information.
I know that the info is in the image but it should be written as well.
Especially to explain that you mean "client ID" in Odoo but "Application (client) ID" in Microsoft Azure.